.github/workflows/release.yaml

```diff
@@ -308,18 +309,22 @@ jobs:
             --submit `
             --version "${version}" `
             --urls "${installer_url}|X64" `
-            --token "${{ secrets.CDRCI_GITHUB_TOKEN }}"
+            --token "$env:WINGET_GH_TOKEN"
 
         env:
           # For gh CLI:
           GH_TOKEN: ${{ github.token }}
+          # For wingetcreate. We need a real token since we're pushing a commit
+          # to GitHub and then making a PR in a different repo.
+          WINGET_GH_TOKEN: ${{ secrets.CDRCI_GITHUB_TOKEN }}
 
       - name: Comment on PR
+        if: ${{ !inputs.dry_run }}
         run: |
           # Find the PR that wingetcreate just made.
           $version = "${{ needs.release.outputs.version }}".Trim('v')
           $pr_list = gh pr list --repo microsoft/winget-pkgs --search "author:cdrci Coder.Coder version ${version}" --limit 1 --json number | `
```

Comment:
Should this backtick at the end be removed too?

Reply:
Backtick is the equivalent of `\` at the end of the line in powershell, so this is intentional